### DISCUSSION TITLE
Beatmap scrubbing in gameplay

### DISCUSSION THREAD

**Discussion by NafeeJ** [ppy/osu#13354] [Ideas] [answered]
Reason: Sometimes I just want to quickly play a specific section of a beatmap, either for practice or fun, without having to play through the entire song for the one part.

Idea: Quickly scrub to a section of a beatmap (McOsu's implementation lets you hold down a key to bring up a scrubber during gameplay)

**Comment by frenzibyte** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/13013.
